pyomo/contrib/appsi/cmodel/src/expression.cpp

```diff
@@ -545,6 +638,12 @@ void MultiplyOperator::propagate_degree_forward(int* degrees, double* values)
 }
 
 
+void ExternalOperator::propagate_degree_forward(int* degrees, double* values)
+{
+  assert (false);
+}
+

```

Comment:
@michaelbynum Similar comment here. Can we add a comment indicating that this will never be necessary or may be necessary later?

Reply:
Done.